src/Keboola/DbWriter/Writer/Snowflake.php

```diff
@@ -63,8 +63,10 @@ public function createConnection(array $dbParams): \PDO
 
     public function createSnowflakeConnection($dbParams): Connection
     {
-        if (getenv('KBC_RUNID')) {
-            $dbParams['runId'] = getenv('KBC_RUNID');
+        $runId = $dbParams['runId'] ?? null;
```

Comment:
Tu by IMHO stacilo nieco take
`$runId = $dbParams['runId'] ?? getenv('KBC_RUNID');`

Ked bude null/false/empty string, tak to cez `if ($runId)` neprejde a nepouzije sa to.

Reply:
je to tam